Directory.Packages.props

```diff
@@ -105,6 +105,7 @@
 
     <!-- Framework packages -->
     <PackageVersion Include="Microsoft.IO.Redist"                                                    Version="6.0.1" />
+    <PackageVersion Include="System.Text.Json"                                                    Version="8.0.5" />
```

Comment:
Could be helpful in future to know why this is here, as we might want to remove it to clean up.

```suggestion
    <!-- Pin version to avoid CVE in System.Text.Json 8.0.4 -->
    <PackageVersion Include="System.Text.Json"                                                    Version="8.0.5" />
```